### DISCUSSION TITLE
Pre-defined name and description texts of missions always get displayed regardless of translation files.

### DISCUSSION THREAD

**Discussion by AzureNocte** [FakeFishGames/Barotrauma#13772] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

I noticed that when 1.4.3.0 comes out, translation of modded mission fails to shows up, it only shows the pre-defined name and description, which I think should not be like this. 
Because these pre-defined mission names and descriptions are for languages that does not have a matching translation file yet(unlike vanilla missions that always have translation available), not intended to replace them. 

Here is the mod I use, it used to work just fine. 
[Undersea Horrors Mission Edition.zip](https://github.com/FakeFishGames/Barotrauma/files/14961109/Undersea.Horrors.Mission.Edition.zip)

### Reproduction steps

1. start modded mission.
2. no translation displayed.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

Other

### -

v1.4.3.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by AzureNocte** [comment]
I suppose this is caused by this fix: 
https://github.com/FakeFishGames/Barotrauma-development/commit/9bfff8aec52cd2224cb1db0bd9681fec12fb8a79

**Comment by Regalis11** [comment]
Thank you for the report! Yep, it seems that fix accidentally changed which texts take priority. [I created a ticket about this](https://github.com/FakeFishGames/Barotrauma/issues/13777), we'll look into it!

**Reply by AzureNocte** [reply]
Thanks!

**Comment by Rokvach** [comment]
Tested the fix for this issue, using the provided mod the translated texts were shown correctly, tested with simplified Chinese. Working correctly, closing dicsussion.
